### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by WilhelmK109**
### In this milestone of the project, I did the following:
- Create a table named owners with id, full_name, and age columns.
- Create a table named species with id, and name columns.
- Remove column species from animals table.
- Modify animals table by adding a PRIMARY KEY, and foreign keys referencing species and owners tables.
- Populate owners table and species table with some data.
- Write query statements to query the database as per project requirements.

Screenshot (126)
Screenshot (127)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -11,4 +11,30 @@ CREATE TABLE animals(
...
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+ALTER TABLE animals
+	ADD PRIMARY KEY (id);
+
```
Comment (thecodechaser) [inline schema.sql:29]: - [ ] In this query you need to remove the column species. Please make sure to fix this query.
  - You can consider this query `ALTER TABLE animals DROP column species;`

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @WilhelmK109👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️

- [ ] The License link in the README file is broken, Please make sure to fix this. It will the README file more professional.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by WilhelmK109** [comment]
@thecodechaser, thanks for the suggestions. I will do the changes and re-submit for a review.

**Review by IngDavidTM — APPROVED** [review]
Hi @WilhelmK109  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by WilhelmK109** [comment]
@IngDavidTM, thank you very much, sir.

**Pull request merged**